libs/shared/src/utils/bridge.utils.ts

```diff
@@ -19,12 +19,18 @@ type InAppOutput = {
   avatar?: string;
   primaryAction?: {
     label: string;
-    url?: string;
+    redirect?: Redirect;
   };
   secondaryAction?: {
     label: string;
-    url?: string;
+    redirect?: Redirect;
   };
+  redirect?: Redirect;
+};
+
+type Redirect = {
+  url: string;
+  target?: UrlTarget;
```

Comment:
can we have this 'Redirect` also exported in shared?

Reply:
Do you mean to just export this type from the `bridge.utils.ts` file?